QuantConnect.DataBento/DataBentoDataProvider.cs

```diff
@@ -113,64 +86,35 @@ public DataBentoProvider(string apiKey)
         private void Initialize()
         {
             Log.Trace("DataBentoProvider.Initialize(): Starting initialization");
-
             _subscriptionManager = new EventBasedDataQueueHandlerSubscriptionManager();
             _subscriptionManager.SubscribeImpl = (symbols, tickType) =>
             {
                 Log.Trace($"DataBentoProvider.SubscribeImpl(): Received subscription request for {symbols.Count()} symbols, TickType={tickType}");
```

Comment:
Never use `.Count()` with `IEnumerable()`.
It's really spamming Log, The Lean already has the same one.
We can remove it.